ipython_examples/TransitTimingVariations.ipynb

```diff
@@ -101,7 +101,7 @@
    "outputs": [],
    "source": [
     "A = np.vstack([np.ones(N), range(N)]).T\n",
-    "c, m = np.linalg.lstsq(A, transittimes)[0]"
+    "c, m = np.linalg.lstsq(A, transittimes, rcond=None)[0]"
```

Comment:
Seems like the tests use a version that doesn't support this yet: https://travis-ci.com/github/Findus23/rebound/jobs/481686474#L897-L903

Reply:
What was the issue without `rcond`?

Reply:
Just the warning message? Or was there an issue with the result?

Reply:
The result is fine.
It's just that numpy is warning:
```
<ipython-input-4-5c626cb5e957>:2: FutureWarning: `rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
  c, m = np.linalg.lstsq(A, transittimes)[0]
```
https://numpy.org/doc/stable/reference/generated/numpy.linalg.lstsq.html

Reply:
But the tests fail...

Reply:
I guess the tests use a numpy version where None wasn't yet a valid parameter.
I switched it now to rcond=-1 which should keep the old behaviour and should also be a valid parameter in the old version.